packages/components/cascader/src/index.vue

```diff
@@ -77,7 +77,14 @@
           </template>
         </el-input>
 
-        <div v-if="multiple" ref="tagWrapper" :class="nsCascader.e('tags')">
+        <div
+          v-if="multiple"
+          ref="tagWrapper"
+          :class="nsCascader.e('tags')"
+          :style="{
+            zIndex: tooltipShow ? 1 : 0,
+          }"
```

Comment:
## My solution

When I looked into this issue, changing the teleport attribute of `el-tooltip` to true solved the issue. (or how to get teleport value as props)

By the way, the style change is inevitable because the `el-tag` is listed differently.

We need input from people to find the right solution.




```vue
// cascader/src/index.vue, line 96
<el-tooltip
  :teleported="true" // or teleported props
  :disabled="popperVisible || !collapseTagsTooltip"
  :fallback-placements="['bottom', 'top', 'right', 'left']"
  placement="bottom"
  effect="light"
/>
```

Reply:
I think if you set teleported true, the dom will be under the body instead of tags and the style will also be inconsistent. 
I just want the hovered item's zIndex higher than the normal select

Reply:
Just change the teleported property and change the style like below.

```scss
// cascader.scss
@include b(cascader) {
  @include e(collapse-tags) {
    white-space: normal;
    z-index: getCssVar('index-normal');
    display: block; // here
    align-items: center;
    flex-wrap: wrap;
  }

  @include e(collapse-tag) {
    line-height: inherit;
    height: inherit;
    display: block; // here
    margin: 2px 0 2px 6px; // here
  }
}
```